pkg/interceptors/client/logging.go

```diff
@@ -0,0 +1,81 @@
+package client
```

Comment:
I would think this is a server interceptor, since it only gets attached to server instances.

Reply:
I can move it!